ansible_collections/arista/avd/roles/eos_designs/tasks/main.yml

```diff
@@ -27,7 +26,7 @@
   run_once: true
 
 - name: Set eos_designs facts
-  tags: [build, provision, facts]
+  when: avd_eos_designs_facts | default(true)
```

Comment:
cut this as an option

Reply:
completed in [bc8d19f](https://github.com/aristanetworks/avd/pull/4427/commits/bc8d19f180681751b75665635a662551a47a2dca)